src/main/java/uk/gov/gchq/palisade/service/SimpleConnectionDetail.java

```diff
@@ -55,26 +58,29 @@ public String createConnection() {
     }
 
     @Override
+    @Generated
     public boolean equals(final Object o) {
```

Comment:
This method needs updating to use the instanceOf methods, so a delete and re-generate should do the trick

Reply:
Done